src/Query/Query.ts

```diff
@@ -36,7 +36,7 @@ export class Query implements IQuery {
     private _ignoreGlobalQuery: boolean = false;
 
     private readonly hideOptionsRegexp =
-        /^(hide|show) (task count|backlink|priority|cancelled date|created date|start date|scheduled date|done date|due date|recurrence rule|edit button|postpone button|urgency|tags|depends on|id|on completion)/i;
+        /^(hide|show) (task count|backlink|priority|cancelled date|created date|start date|scheduled date|done date|due date|recurrence rule|edit button|postpone button|urgency|tags|depends on|id|on completion|children)/i;
```

Comment:
Wow, this has grown a lot. I would need we probably need an abstraction here to add show/hide options. This is obviously of the scope of this PR.

Reply:
Yes - definitely agree.

We will need to be careful not to repeat the unintended breaking of existing queries made when the equivalent regex for grouping options was replaced:

- https://github.com/obsidian-tasks-group/obsidian-tasks/issues/1923